aptos-move/framework/aptos-framework/sources/block.move

```diff
@@ -154,7 +152,7 @@ module aptos_framework::block {
     }
 
     #[test(aptos_framework = @aptos_framework, account = @0x123)]
-    #[expected_failure(abort_code = 0x50002)]
+    #[expected_failure(abort_code = 0x50003)]
```

Comment:
All error codes in system_addresses were shifted up from removing 0-indexed error code.